dlt/extract/incremental/__init__.py

```diff
@@ -284,11 +300,53 @@ def parse_native_representation(self, native_value: Any) -> None:
             self._primary_key = merged._primary_key
             self.allow_external_schedulers = merged.allow_external_schedulers
             self.row_order = merged.row_order
+            self._lag = merged.lag
             self.__is_resolved__ = self.__is_resolved__
         else:  # TODO: Maybe check if callable(getattr(native_value, '__lt__', None))
             # Passing bare value `incremental=44` gets parsed as initial_value
             self.initial_value = native_value
 
+    def _apply_lag(self, value: TCursorValue) -> TCursorValue:
+        if self._lag is None:
```

Comment:
we should also skip functions that are not min / max and if we do so, we warn the user